lib/routes/kelownacapnews/news.ts

```diff
@@ -0,0 +1,85 @@
+import { DataItem, Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/:type',
+    categories: ['new-media'],
+    example: '/kelownacapnews/local-news',
+    parameters: { type: 'Type of news' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.kelownacapnews.com/:type'],
+            target: '/:type',
+        },
+    ],
+    name: 'News',
+    maintainers: ['hualiong'],
+    url: 'www.kelownacapnews.com',
+    description: `\`type\` is as follows:
+  
+| News type     | Value         | News type    | Value        |
+| ------------- | ------------- | ------------ | ------------ |
+| News          | news          | Sports       | sports       |
+| Local News    | local-news    | Business     | business     |
+| Canadian News | national-news | Trending Now | trending-now |
+| World News    | world-news    | Opinion      | opinion      |
+| Entertainment | entertainment |              |              |`,
+    handler: async (ctx) => {
+        const type = ctx.req.param('type');
+        const baseURL = 'https://www.kelownacapnews.com';
+
+        const response = await ofetch(`${baseURL}/${type}`);
+        const $ = load(response);
+
+        const list = $('.media')
+            .toArray()
+            .map((item): DataItem => {
+                const a = $(item);
+                return {
+                    title: a.find('.media-heading').text(),
+                    pubDate: timezone(parseDate(a.find('.media-links time').attr('datetime')!), +8),
```

Comment:
The 'Z' in the `datetime` indicates that the time string is in Zulu time, which should not include a timezone offset.